crypto/mem_clr.c

```diff
@@ -60,6 +60,7 @@
 #include <string.h>
 #include <openssl/crypto.h>
 
+extern unsigned char cleanse_ctr;
```

Comment:
I don't get it. The error message suggests that problem line reads `extern unsigned char cleanse_ctr = 0;`. I have no problem with compiler issuing warning in such case, and even concur that it should be addressed. But original code reads `unsigned char cleanse_ctr = 0;`, i.e. there is no extern. And if compiler complains about that, then it's more likely to be a bug in warning system...